### PR TITLE
release: bump to v0.7.0

### PR DESCRIPTION
**Pull request by TennyZhuang**
Signed-off-by: TennyZhuang <zty0826@gmail.com>

### CODE REVIEW

**Review by wangrunji0408 — COMMENTED** [review]
#89 added a Sync constraint to the error type, breaking the compatibility.
So I think bumping to 0.7 would be better.

File: Cargo.toml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 members = ["sqllogictest", "sqllogictest-bin", "examples/*", "tests"]
 
 [workspace.package]
-version = "0.6.4"
+version = "0.6.5"
```
Comment (TennyZhuang) [inline Cargo.toml:5]:
```suggestion
version = "0.7.0"
```

**Review by wangrunji0408 — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.6.5] - 2022-11-14
```
Comment (wangrunji0408) [inline CHANGELOG.md:9]: missed 🤪

Comment (TennyZhuang) [reply CHANGELOG.md:9]: 😭

**Pull request merged**